Source/Obstacle/Obstacle.h

```diff
@@ -13,7 +13,8 @@
 enum
 {
 	maxDefence = 100,
-	minDefence = 0
+	minDefence = 0,
+	defaultDefece = 100
```

Comment:
Small thing default Defense is spelled wrong. Why does an obstacle have a default defense of 100? Have we set a max or a min defense?

Reply:
I was thinking that we might have a default value in the future, so I added that to be changed in the future for a default value. I will just make it max defense.